sdk/storage/azure_storage_blob/src/models/blob_properties.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core::headers::Headers;
+
+/// Properties of an Azure Storage blob.
+///
+#[derive(Clone, Default, Debug)]
+pub struct BlobProperties {
+    pub access_tier_inferred: String,
+    pub access_tier: String,
+    pub blob_type: String,
+    pub content_length: String,
+    pub content_md5: String,
+    pub content_type: String,
+    pub creation_time: String,
+    pub etag: String,
+    pub last_modified: String,
```

Comment:
Why is this a string and not a `DateTimeOffset` which we use for times elsewhere.

Reply:
wrt this comment, is there a currently accepted way to convert from RFC1123 strings (as returned by the service) ex:
`creation_time: Some("Sat, 01 Feb 2025 00:17:37 GMT"` to a `OffsetDateTime` object? I know you can provide a custom pattern query to [parse](https://docs.rs/time/latest/time/struct.OffsetDateTime.html#method.parse), but I assume that if we have `OffsetDateTime` elsewhere we may have some common utility for this?